.env.example

```diff
@@ -0,0 +1,20 @@
+# The ID of your GitHub App
+APP_ID=
```

Comment:
For these entries, you want to put in sample values that match the expected type (ie. if it wants an int, then put in 1234; string: example_id, etc)

Reply:
Done, thanks!